torch/csrc/jit/OVERVIEW.md

```diff
@@ -71,25 +71,27 @@ Sections start with a reference to the source file where the code related to the
 
 ## Modules ##
 
-[script/module.h](../script/module.h)
+[api/module.h](api/module.h)
 
 At the top level, all TorchScript programs are represented as a Module. Modules contain:
 
 * named Parameters - tensors used in training such as `weight` or `bias`
-* named Methods - functions that can be run on the module such as `forward`
+* named Buffers - tensors that are part of the training state of a module but do not appear in module.parameters() and do not participate in gradient descent.
```

Comment:
Buffers are the same as tensor attributes, they should probably be in the same bullet

Reply:
that's more an implementation detail; since the api has `buffers()` and `named_buffers()` I thought we might want to call them out specifically.